.vscodeignore

```diff
@@ -6,3 +6,18 @@ erlang.tmbundle/Commands
 erlang.tmbundle/Macros
 erlang.tmbundle/Preferences
 erlang.tmbundle/Snippets
+
+.vscode/**
+**/*.ts
+**/*.map
+.gitignore
+**/tsconfig.json
+**/tsconfig.base.json
+contributing.md
+.travis.yml
+client/node_modules/**
+!client/node_modules/vscode-jsonrpc/**
+!client/node_modules/vscode-languageclient/**
+!client/node_modules/vscode-languageserver-protocol/**
+!client/node_modules/vscode-languageserver-types/**
+!client/node_modules/semver/**
```

Comment:
Those last five lines should be recognised in the `client/node_modules/**` line, and so are redundant.

Reply:
Aren't those negations?